src/main/java/WriteDatasetTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+import core.ZOSConnection;
+import zosfiles.ZosDsn;
+
+public class WriteDatasetTest {
+
+    public static void main(String[] args) throws Exception {
+        String hostName = "XXX";
+        String port = "XXX";
+        String userName = "XXX";
+        String password = "XXX";
+        String datasetMember = "XXX";
+
+        ZOSConnection connection = new ZOSConnection(hostName, port, userName, password);
+
+        WriteDatasetTest.writeToDsnMember(connection, datasetMember, "NEW CONTENT\nTHE SECOND LINE UPDATED");
+    }
+
+    private static void writeToDsnMember(ZOSConnection connection, String datasetMember, String content) {
+        ZosDsn.writeDsn(connection, datasetMember, content);
```

Comment:
what are your thoughts on using new ZosDsn() in a future refactor instead of static method calls? I think we will need to do this to add mock tests.